models/src/main/kotlin/io/provenance/api/models/cee/approve/ApproveContractExecutionResponse.kt

```diff
@@ -0,0 +1,8 @@
+package io.provenance.api.models.cee.approve
+
+import io.provenance.api.models.p8e.TxResponse
+
+data class ApproveContractExecutionResponse(
+    val envelopeState: String,
```

Comment:
Can you have this type as ByteArray as well?

Reply:
updated the name as requested in slack